src/components/Footer/Footer.tsx

```diff
@@ -0,0 +1,90 @@
+import React from 'react';
+import { Todo } from '../../types/Todo';
+import cn from 'classnames';
+import { FilterStatus } from '../../types/enums';
+
+type Props = {
+  allTodos: Todo[];
+  selectedValue: (value: FilterStatus) => void;
+  onSelect: FilterStatus;
+  handleDeleteCompleted: () => void;
+};
+
+export const Footer: React.FC<Props> = ({
+  allTodos,
+  selectedValue,
+  onSelect,
+  handleDeleteCompleted,
+}) => {
+  const completedTodosLength = allTodos.filter(todo => todo.completed).length;
+  const isHidden = completedTodosLength === 0;
+
+  const valueForFilter = [
+    {
+      href: '#/',
+      dataCy: 'FilterLinkAll',
+      buttonTitle: 'All',
+      status: FilterStatus.All,
+    },
+    {
+      href: '#/active',
+      dataCy: 'FilterLinkActive',
+      buttonTitle: 'Active',
+      status: FilterStatus.Active,
+    },
+    {
+      href: '#/completed',
+      dataCy: 'FilterLinkCompleted',
+      buttonTitle: 'Completed',
+      status: FilterStatus.Completed,
+    },
+  ];
+
+  function notCompletedTodo() {
```

Comment:
It is important to stick to one way of creating functions everywhere
```suggestion
  const notCompletedTodo = () => {
```